File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.py
def GCD(a,b):
    while(b):
        a,b = b,a%b
    return a
def LCM(a,b):
    result = (a*b)//GCD(a,b)
    return result

a, b = map(int, input().split())
print(GCD(a,b))
print(LCM(a,b))

File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br =  new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        
        int n = Integer.parseInt(st.nextToken());
        int m = Integer.parseInt(st.nextToken());
        int lcm = findlcm(n,m);
        int gcd = findgcd(n,m);
        System.out.println(gcd);
        System.out.println(lcm);
        
    }
    public static int findgcd(int n,int m){
        while(m!=0){
            int temp = m;
            m = n%m;
            n = temp;
        }
        return n;

    }
    public static int findlcm(int n, int m){
        return (n*m)/findgcd(n, m);

    }

}